_posts/Getting Started/2020-07-14-create-flink-project-on-streaming-data-platform.md

```diff
@@ -20,39 +20,33 @@ This is a general guide for creating and setting up Flink job on Dell EMC Stream
 ![Analytics-Project]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/analytics-project.png)
 **Note:** If you cannot log in to the Dell EMC Streaming Data Platform, please use ```kubectl get ingress -A``` to find all ingress and add to your **/etc/hosts** file.  
 
-**2.** Click **Create Project** button to start with a new project by typing the project name, description and required volume size. This will also automatically create a scope or namespace for your project in Pravega. 
-![Create-Project]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/create-project.png)
+**B.** Click **Create Project** button to start with a new project by typing the project name, description and required volume size. This action will also automatically create a scope or namespace for your project in Pravega. 
+![Create-Project]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/create-project.png)      
 
-##### C. Create Flink Clusters
-**1.** To create a Flink cluster using the Dell EMC Streaming Data Platform UI, navigate to **Analytics -> *project-name* > Flink Clusters**.  
-![Flink-Cluster]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/flink-cluster.png)
-
-**2.** Click **Create Flink Cluster** and complete the cluster configuration fields.
-![Create-Flink-Cluster]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/create-flink-cluster.png)
 
-##### D. Upload artifact to SDP
-###### Option I: Using the Dell EMC Streaming Data Platform UI
-**1.** Navigate to **Analytics > Analytics Projects > *project-name* > Artifacts**
+##### 3. Upload artifact to SDP
+###### Option A: Using the Dell EMC Streaming Data Platform UI
+**I.** Navigate to **Analytics > Analytics Projects > *project-name* > Artifacts**
 ![Artifact-UI]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/artifact-ui.png)
 
-**2.** Click **Upload Artifact**. Under the **General** section, specify the **Maven Group**, the **Maven Artifact**, and enter the **Version number**. Under the **Artifact** File section, browse to the JAR file on your local machine and select the file to upload to the repository. 
+**II.** Click **Upload Artifact**. Under the **General** section, specify the **Maven Group**, the **Maven Artifact**, and enter the **Version number**. Under the **Artifact** File section, browse to the JAR file on your local machine and select the file to upload to the repository. 
 ![Upload-Artifact]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/upload-artifact.png)
 
-###### Option II: Using the Gradle Build Tool
-**1.** Make the Maven repo in SDP available to your development workstation.
+###### Option B: Using the Gradle Build Tool
+**I.** Make the Maven repo in SDP available to your development workstation.
 ```
 kubectl port-forward service/repo 9090:80 --namespace project-name
```

Comment:
`kubectl port-forward` is actually not the preferred approach. Instead, have Gradle publish to the Maven repo service that is exposed as an ingress. You might be interested in https://github.com/pravega/flink-tools/blob/master/scripts/publish.sh but this may be too complicated for an intro blog.

Reply:
Because this post refers to some instructions in workshop-sample. In order to keep consistency, I will leave as `kubectl port-forward` right now. But I will definitely update the instruction once I test the scripts and update the README for workshop-sample.

Reply:
I break the `publish.sh` into a few steps in which users may easily follow the instructions.  The instructions teach how to get host and IP from ingress, add to the `hosts` file, and use `build.gradle`.  Feel free to give me some suggestions.